packages/editor/src/client/i18n.ts

```diff
@@ -40,12 +37,4 @@ i18n.use(LanguageDetector).use(initReactI18next).init({
   }
 })
 
-// TODO: how to handle other date amd time formats
-registerLocale('de', dateFnsDe)
```

Comment:
@penk85 why did you remove this?

Reply:
@roosnic1 was a merge conflict and I went with master version